src/pages/index.js

```diff
@@ -0,0 +1,327 @@
+import lightPencilIcon from '../images/whitepencil.svg';
+import plusIcon from '../images/button.svg';
+import Api from '../utils/Api.js';
+
+document.getElementById('header-logo').src = logo;
+document.getElementById('profile-avatar').src = avatar;
+document.getElementById('profile-post-icon').src = plusIcon;
+document.getElementById('profile-avatar-edit-icon').src = lightPencilIcon;
+document.getElementById('profile-edit-icon').src = pencilIcon;
+
+const api = new Api({
+  baseUrl: 'https://around-api.en.tripleten-services.com/v1',
+  headers: {
+      authorization: 'd9b462f6-0ab1-4657-9af2-202693166e25',
+      'Content-Type': 'application/json'
+  }
+});
+
+
+
+  // Edit Profile Modal
+  const profileName = document.querySelector('.profile__name');
+  const profileDescription = document.querySelector('.profile__description');
+  const editButton = document.querySelector('.profile__edit-btn');
+  const editProfileModal = document.querySelector('#edit-profile');
+  const editProfileCloseButton = editProfileModal.querySelector('#edit-profile-close-btn');
+  const editModalNameInput = editProfileModal.querySelector('#profile-name-input');
+  const editModalDescriptionInput = editProfileModal.querySelector('#profile-description-input');
+  const editProfileForm = document.forms['profile-form'];
+
+  // New Post Modal
+  const newPostButton = document.querySelector('.profile__post-btn');
+  const newPostModal = document.querySelector('#add-card-modal');
+  const newPostCloseButton = newPostModal.querySelector('#new-post-close-btn');
+  const newPostForm = document.forms['new-post'];
+  const newPostLinkInput = newPostModal.querySelector('#add-card-link-input');
+  const newPostCaptionInput = newPostModal.querySelector('#add-card-caption-input');
+  const cardSubmitBtn = newPostModal.querySelector(".modal__submit-btn");
+
+  // Preview Modal
+  const previewModal = document.querySelector('#preview-modal');
+  const previewModalImage = previewModal.querySelector('.modal__image');
+  const previewModalCaption = previewModal.querySelector('.modal__caption');
+  const previewCloseButton = previewModal.querySelector('.modal__close-btn_type_preview');
+
+  //Avatar Modal
+  const editAvatarModal = document.getElementById('edit-avatar-modal');
+  const editAvatarCloseButton = document.querySelector('#edit-avatar-close-btn');
+  const editAvatarForm = editAvatarModal.querySelector('form');
+  const editAvatarLinkInput = editAvatarModal.querySelector('#edit-avatar-link-input');
+  const avatarSubmitBtn = editAvatarModal.querySelector('.modal__submit-btn');
+  const profileAvatar = document.getElementById('profile-avatar');
+
+  function closeAvatarModal() {
+    closeModal(editAvatarModal);
+  }
+
+  const openAvatarModal = () => {
+    resetValidation(editAvatarForm, [editAvatarLinkInput]);
+    openModal(editAvatarModal);
+  };
+
+
+  function handleEditAvatarSubmit(evt) {
+    evt.preventDefault();
+
+    if (!editAvatarForm.checkValidity()) {
```

Comment:
`    if (!editAvatarForm.checkValidity()) {
`

It's not needed because you have **validation** that disables the button if the form is invalid